jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/handler/PodTemplateHandlerTest.java

```diff
@@ -91,15 +86,15 @@ public void podWithoutVolumeTemplateHandlerTest() {
         PodTemplateSpec podTemplateSpec = podTemplateHandler.getPodTemplate(config, null, images);
 
         //Assertion
-        assertEquals("test-account", podTemplateSpec.getSpec().getServiceAccountName());
-        assertTrue(podTemplateSpec.getSpec().getVolumes().isEmpty());
-        assertNotNull(podTemplateSpec.getSpec().getContainers());
-        assertEquals("test-app", podTemplateSpec.getSpec()
-                .getContainers().get(0).getName());
-        assertEquals("docker.io/test:latest", podTemplateSpec.getSpec()
-                .getContainers().get(0).getImage());
-        assertEquals("IfNotPresent", podTemplateSpec.getSpec()
-                .getContainers().get(0).getImagePullPolicy());
+        assertThat( podTemplateSpec.getSpec().getServiceAccountName()).isEqualTo("test-account");
+        assertThat(podTemplateSpec.getSpec().getVolumes()).isEmpty();
+        assertThat(podTemplateSpec.getSpec().getContainers()).isNotNull();
+        assertThat( podTemplateSpec.getSpec()
+                .getContainers().get(0).getName()).isEqualTo("test-app");
+        assertThat(podTemplateSpec.getSpec()
+                .getContainers().get(0).getImage()).isEqualTo("docker.io/test:latest");
+        assertThat(podTemplateSpec.getSpec()
+                .getContainers().get(0).getImagePullPolicy()).isEqualTo("IfNotPresent");
```

Comment:
Please chain assertions here. Try to do assertion using one `assertThat` statement like done in other places.